tests/end2end/configs/bgc_tasknn_bert.jsonnet

```diff
@@ -0,0 +1,141 @@
+{
+  dataset_reader: {
+    type: 'bgc',
+    max_instances: 100,
+    token_indexers: {
+      x: {
+        type: 'pretrained_transformer',
+        model_name: 'bert-base-uncased',
+      },
+    },
+    tokenizer: {
+      type: 'pretrained_transformer',
+      max_length: 512,
+      model_name: 'bert-base-uncased',
+    },
+  },
```

Comment:
Specific dataset reader with tokenizer and token_indexer. See allennlp guide to understand their roles.